### PR TITLE
Removed duplicate includes of <osgUtil/LineSegmentIntersector>

### PR DESCRIPTION
**Pull request by beachwalker**
Removed a duplicated inclusion of <osgUtil/LineSegmentIntersector> in EarthManipulator.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -138,6 +138,7 @@ FIND_PACKAGE(SilverLining QUIET)
 FIND_PACKAGE(Triton QUIET)
 
 FIND_PACKAGE(Protobuf QUIET)
+SET (PROTOBUF_USE_DLLS FALSE CACHE BOOL "Set this to true if Protobuf is compiled as dll")
```
Comment (filnet) [inline CMakeLists.txt]: seems unrelated to the PR title ?

**Comment by filnet** [comment]
The commit history is a bit messy and seems to contain unrelated changes.

**Comment by beachwalker** [comment]
Jep, sorry. Was a little bit in a hurry.

**Comment by jasonbeverage** [comment]
Thanks for this, I went ahead and made the change manually to avoid bringing in unrelated changes.  It's removed in c50874a7defe0c911b49bb8665bc339eb2689452
